src/inet/EndPointBasis.h

```diff
@@ -112,63 +80,18 @@ class DLL_EXPORT EndPointBasis : public InetLayerBasis
 #endif                  // INET_CONFIG_ENABLE_TCP_ENDPOINT
     };
 
-    enum
+    enum class LwIPEndPointType : uint8_t
     {
-        kLwIPEndPointType_Unknown = 0,
-
-        kLwIPEndPointType_Raw = 1,
-        kLwIPEndPointType_UDP = 2,
-        kLwIPEndPointType_UCP = 3,
-        kLwIPEndPointType_TCP = 4
-    };
-
-    uint8_t mLwIPEndPointType;
-
-    void DeferredFree(chip::System::Object::ReleaseDeferralErrorTactic aTactic);
+        Unknown = 0,
+        Raw     = 1,
+        UDP     = 2,
+        UCP     = 3,
+        TCP     = 4
```

Comment:
nit: `kRaw`, `kUDP`, etc.

Reply:
Either way I get someone asking for the other :-)

I'll touch this again in a future step toward #7715.